nautobot/extras/templates/extras/plugin_detail.html

```diff
@@ -32,7 +28,7 @@
         {% endblock buttons %}
     </div>
     {% block masthead %}
-        <h1>{% block title %}{{ app_data.name | bettertitle }}{% endblock %}</h1>
+        <h1>{% block title %}{% render_title %}{% endblock %}</h1>
     {% endblock masthead %}
```

Comment:
Any chance we can remove the overridden `block header` from this file and only override `block buttons` instead?

Reply:
Makes sense, I guess that's because an AppConfig isn't a Model. Thanks for checking.